app/api-v1/services/recipeService.js

```diff
@@ -21,7 +21,18 @@ async function getRecipes() {
   return getRecipesDb()
 }
 
+async function getRecipeByID(id) {
+  const recipeResult = await getRecipeByIDdb(id)
+  if (recipeResult.length === 0) {
+    throw new ItemNotFoundError({ message: 'Item not found', service: 'order' })
+  } else {
+    const result = recipeResult[0]
+    return { statusCode: 200, result }
```

Comment:
just return `result` here. No need to propagate the status code. This function may be used by something that's not immediately repsonding to a request